processes/community/onboarding/README.md

```diff
@@ -5,127 +5,89 @@ Complete the instructions below, and you will be enrolled shortly.
 
 **This process is only available for pre-approved accounts. To request approval, contact [support@opes.one](mailto:support@opes.one)**
 
-**Note**: *These instructions are deprecated and will expire on September 1st, 2019. A new enrollment process will be implemented at that time.*
+**Note**: *These instructions are deprecated and will expire on October 1st, 2019. A new enrollment process will be implemented at that time.*
```

Comment:
Does this mean they are already expired?

Reply:
Yes, but maybe not.

@RocketDapps This is the public-facing copy of the same file from `o1.documentation`. Please comment on whether the files are out of date, or whether we should extend their expiration.

Reply:
@ladybug I believe Brendan is on vacay until Tues, so make sure you follow up with him after tomorrow.  If you don't hear from him, ping me.

Reply:
October 1st, 2019 is in the past?